testing/sdk_metrics.py

```diff
@@ -0,0 +1,36 @@
+'''Utilities relating to checking services' metrics'''
```

Comment:
I'd like this to say something about what's being tested here.
eg "verifies metrics functionality by fetching metrics from the DC/OS metric component"